apps/idos-data-dashboard/src/core/idos.tsx

```diff
@@ -136,7 +136,7 @@ export const Provider = ({ children }: PropsWithChildren) => {
         address: userAddress,
         publicKey,
         async reset() {
-          await sdk.reset({ enclave: true });
+          await sdk.reset();
```

Comment:
Do we still use this? A quick grep told me "no":
```bash
$ git grep -E 'useIdOS\(' apps/idos-data-dashboard/
apps/idos-data-dashboard/src/components/proof-of-personhood.tsx:40:  const { address } = useIdOS();
apps/idos-data-dashboard/src/core/idos.tsx:40:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/components/credential-details.tsx:23:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/components/delete-credential.tsx:36:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/index.tsx:27:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/index.tsx:135:  const { hasProfile } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/shared/index.ts:6:  const { sdk, address, publicKey } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/credentials/shared/index.ts:31:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/index.tsx:41:  const { address } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/wallets/components/add-wallet.tsx:39:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/wallets/components/delete-wallet.tsx:27:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/wallets/index.tsx:26:  const { sdk } = useIdOS();
apps/idos-data-dashboard/src/routes/dashboard/wallets/index.tsx:95:  const { hasProfile } = useIdOS();
```

Reply:
I'd rather remove it to reduce having unnecessary things around, but I don't oppose. :+1: